### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by Rantelo**
Please, follow the tutorial in /guides and leave a comment if you find something weird.

### CODE REVIEW

File: docs_source/src/containers/Guides.js [hunk 1]
```diff
@@ -0,0 +1,471 @@
...
+    {` Wizardo: run page_generator - DONE!
+  create: pages/products.html `}
+  </Highlight>
+  <p>And you can check the file creation in your file system. It should look something like the following:</p>
+<Highlight className='bash'>
+     {` website/
+   ├ .wizardo/
+   │ ├─ page_generator.config.json
+   │ └─ templates/
+   │    └─ index.html
```
Comment (uquintana) [inline docs_source/src/containers/Guides.js]: This should be `___page_name___.html` as you renamed this file before.

**Comment by iburgos72** [comment]
I think that is good idea white a sublist in the lateral menu with all the subtitles. something like this:

> - First steps
>   1. Installation
>   2. Wizardo init
>   3. Should I commit this folder in git?
>   4. (...)
> - Create a generator
>   1. (...)
>   2. (...)
>   (...)

with the sublist the people can know all the content in the guide

**Comment by iburgos72** [comment]
you have a link to `Run the generator section.` but this section doesn't exist
 [ ] delete the link or create the section

File: docs_source/src/App.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        <Content style={{minHeight: 'calc(100vh - 70px)'}}>
+          <Router>
+            <Switch>
+              <Route path='/guides' component={Guides} />
+              <Routes />
+            </Switch>
+          </Router>
+        </Content>
+        <Footer style={{ textAlign: 'center' }}>
+          Wizardo ©2018 Created by Intelimétrica
```
Comment (iburgos72) [inline docs_source/src/App.js]:
```suggestion
          2018
```
can you change the year by (new Date).getFullYear()? with the function is not necessary update the year in 2 months

File: docs_source/src/containers/Guides.js [hunk 3]
```diff
@@ -0,0 +1,471 @@
...
+</section>
+
+<section id='modifiers'>
+  <h1>Modifiers</h1>
+  <p>Modifiers are the way of looking for a pattern in existing files and adding new content to them.<br />
+    They work with <InlineCode>Regular Expressions</InlineCode> so if you are unfamiliar with those, we recommend you to go take a look at them in <a href="https://regexone.com/" target="_blank">regexone.com</a> or the resource of your choice.
+  </p>
+  <h2>1. Modifiers structure</h2>
+  <p>The content of <InlineCode>modifiers</InlineCode> key in the configuration file is made of an <InlineCode>array</InlineCode> of <InlineCode>path_to_file</InlineCode>, <InlineCode>regex</InlineCode> and <InlineCode>text_to_insert</InlineCode>.</p>
+  <p> - <InlineCode>path_to_file</InlineCode> indicates the path from the root of the project to the file that the modifier is goint to work on.<br />
```
Comment (ArmandoCardoso) [inline docs_source/src/containers/Guides.js]: Typo: goint -> going

**Pull request merged**